Wire-iOS/Sources/UserInterface/Camera/CameraController.swift

```diff
@@ -283,7 +283,7 @@ private extension AVCaptureVideoOrientation {
     /// then the interface orientation. Must be called on the main thread.
     static var current: AVCaptureVideoOrientation {
         let device = UIDevice.current.orientation
-        let ui = UIApplication.shared.statusBarOrientation
+        let ui = UIApplication.shared.windows.first!.windowScene!.interfaceOrientation
```

Comment:
Here we might go for optional chaining for more safe unwrapping with unknown orientation, and same for the other places where it's used.
```suggestion
        let ui = UIApplication.shared.windows.first?.windowScene?.interfaceOrientation ?? .unknown
```

Reply:
Done!